arch/vocab.md

```diff
@@ -0,0 +1,34 @@
+# About
+A registry of vocabulary shared among all projects.
+
+# Abstract
+These concepts are not tied to any specific context.
+
+* Offer: to provide a service
+	- to present an option of satisfying a requirement
+	- [Spec](https://schema.org/Offer)
+	- i.e. selling tickets for an event
+
+* Section: a part of a partition
+	- **alias of** Part
+	- [More on partitions](https://en.wikipedia.org/wiki/Partition_of_a_set)
+	- i.e. one slice of cake
+
+# Academic
+* Course: 
+	- **alias of** Curricular Unit
+	- [Spec](https://schema.org/Course)
+	- i.e. Software Laboratory
+
+* Class: an instance of Course in a given Semester
+	- **alias of** Course Offer
+	- **alias of** Course Instance
+	- [Spec](https://schema.org/CourseInstance)
+	- i.e. Software Laboratory, v1920
+
+* Class Section: a non-empty sub-set of a Class
```

Comment:
I don't think a `Class` is a set, so it is strange to define class section as a sub-set.

Reply:
Isn't there a schema.org concept that matches this?